src/Services/SubscriptionService.php

```diff
@@ -203,17 +203,18 @@ public function issueUpcomingInvoice($subscriptionId, $invoiceId)
     /**
      * @param string $subscriptionId
      * @param array|JsonSerializable|SubscriptionChangeItems $data
+     * @param array|ArrayObject $params
      *
      * @throws DataValidationException if input data is not valid
      *
      * @return Subscription
      */
-    public function changeItems($subscriptionId, $data)
+    public function changeItems($subscriptionId, $data, $params = [])
     {
         return $this->client()->post(
             $data,
             'subscriptions/{subscriptionId}/change-items',
-            ['subscriptionId' => $subscriptionId]
+            ['subscriptionId' => $subscriptionId] + $params
```

Comment:
This operator does not work with `ArrayAccess` (allowed according phpdoc)

Reply:
@slavcodev , thank you! I'll fix in upcoming release

Reply:
[Here is the fix](https://github.com/Rebilly/rebilly-php/pull/531)